.github/scripts/synchronize_jira_xray.js

```diff
@@ -0,0 +1,320 @@
+const axios = require("axios");
+const fs = require("fs");
+const FormData = require('form-data');
+
+const JIRA_USER = process.env.JIRA_USER;
+const JIRA_TOKEN_TEST = process.env.JIRA_TOKEN_TEST;
+const CLIENT_ID = process.env.CLIENT_ID;
+const CLIENT_SECRET = process.env.CLIENT_SECRET;
+
+const XRAY_API_URL =
+  "https://xray.cloud.getxray.app/api/v2/import/feature?projectKey=MON";
+const GRAPHQL_URL = "https://xray.cloud.getxray.app/api/v2/graphql";
+
+async function get_xray_token(clientId, clientSecret) {
+  const data = { client_id: clientId, client_secret: clientSecret };
+  try {
+    const response = await axios.post(
+      "https://xray.cloud.getxray.app/api/v1/authenticate",
+      data,
+      {
+        headers: { "Content-Type": "application/json" },
+      }
+    );
+
+    if (response.status === 200) {
+      const token = response.headers["x-access-token"];
+      if (token) {
+        console.log("Authentication successful");
+        return token;
+      } else {
+        console.log(
+          "Authentication failed. Token not found in the response headers."
+        );
+      }
+    } else {
+      console.log(`Authentication failed with status code: ${response.status}`);
+    }
```

Comment:
use return early pattern instead of nested `if` :  https://medium.com/swlh/return-early-pattern-3d18a41bba8
example : 
```
if (response.status !== 200) {
  return;
}
```